fix: record too large error

fraidev opened the pull request:
Got this error testing max_request_size on python client: 

```
mymodule.PyFluvioError: record size (17 bytes), exceeded maximum request size (17 bytes)
```

Should be

```
mymodule.PyFluvioError: record size (17 bytes), exceeded maximum request size ("max_request_size" bytes)
```

sehz reviewed (commented):
Can you check fluvio need to be bumped?

fraidev commented:
> Can you check fluvio need to be bumped?

Sure, I didn't update because I updated already at https://github.com/infinyon/fluvio/pull/4375.

But I also updated here.
